web/viewer-geckoview.css

```diff
@@ -30,7 +30,11 @@
   --dialog-button-bg-color: rgba(12, 12, 13, 0.1);
   --dialog-button-hover-bg-color: rgba(12, 12, 13, 0.3);
 
-  --toolbarButton-download-icon: url(images/toolbarButton-download.svg);
+  --toolbar-bg-color: #f9f9fb;
+  --toolbar-divider-color: #e0e0e6;
```

Comment:
Do we perhaps want to use a slightly darker colour value here, since it's currently quite difficult to distinguish it from the viewer-background?

Reply:
UI/UX provided the colors... I'd prefer keeping these colors for the moment.
And that stuff will probably be changed in a close future.